src/main/java/com/pmm/budgetcalculator/expensing/service/ExpenseService.java

```diff
@@ -34,17 +42,17 @@ public void removeExpenseById(Long id) {
         } else throw new ExpenseNotFoundException(id);
```

Comment:
Although is not incorrect there is unwritten contract that java devs always add {} in ifs, elses, try, catches, loop etc.